src/MaxText/layers/nnx_wrappers.py

```diff
@@ -163,12 +156,6 @@ def lazy_init(fn: Module | tp.Callable[..., tp.Any], *args, **kwargs):
   return fn
 
 
-def current_linen_module() -> linen.Module | None:
-  """Get the current Linen module from the Linen context."""
-  if linen.module._context.module_stack:  # pylint: disable=W0212
-    return linen.module._context.module_stack[-1]  # pylint: disable=W0212
-  return None
-
```

Comment:
Could you double check if all the changes in this file are intentional? I had accidentally added extra changes on the branch I had. I think the changes in this file should just be all the ones for `to_linen_class`

Reply:
I didn't see this one is called. I will just remove it?